src/adhocracy_core/docs/deletion.rst

```diff
@@ -19,11 +19,12 @@ The effect of deleting is as follows:
 * If the frontend attempts to retrieve a *deleted* resource via
   GET, the backend responds with HTTP status code *404 Not Found*, just
   as if the resource had never existed.
-* *Deleted* resources may still be referenced from other
-  resources. If the frontend follows such references it must therefore
+* *Deleted* resources may not be referenced from other
+  resources. If the frontend follows an outdated references it must therefore
   be prepared to encounter *404 Not Found* responses and deal with them
   appropriately (e.g. by silently skipping them or by showing an
   explanation such as "Comment deleted").
```

Comment:
How does this work? Are all resources referencing a deleted resource also deleted? Or are the references removed?